### ISSUE TITLE
[BUG] In Rancher v2.9.0/v2.9.1 w/ Harvester v1.3.2-rc2 "Support" Link Doesn't Link To Support

### ISSUE THREAD

**Issue by irishgordo** [harvester/harvester#6442]
**Describe the bug**
Clicking "Support" through Rancher v2.9.0/v2.9.1 w/ Harvester v1.3.2-rc2 doesn't take Rancher user to Harvester Support, it redirects them back to their Local Cluster Homepage for Rancher

**To Reproduce**
1. Import Harvester into Rancher
2. In Rancher build a custom-role template, with some content:
```
apiVersion: management.cattle.io/v3
displayName: do-some-things
kind: GlobalRole
##############
rules:
  - apiGroups:
      - harvesterhci.io.management
    resources:
      - clusters
    verbs:
      - create
      - delete
      - get
      - list
      - patch
      - update
      - watch
  - apiGroups:
      - harvesterhci.io
    resources:
      - VirtualMachineTemplate
    verbs:
      - create
      - delete
      - get
      - list
      - patch
      - update
      - watch
  - apiGroups:
      - harvesterhci.io
    resources:
      - VirtualMachienInstance
    verbs:
      - create
      - delete
      - get
      - list
      - patch
      - update
      - watch
  - apiGroups:
      - harvesterhci.io
    resources:
      - VirtualMachineImage
    verbs:
      - create
      - delete
      - get
      - list
      - patch
      - update
      - watch
  - apiGroups:
      - harvesterhci.io
    resources:
      - Networks
    verbs:
      - create
      - delete
      - get
      - list
      - patch
      - update
      - watch
  - apiGroups:
      - harvesterhci.io
    resources:
      - ClusterNetworks
    verbs:
      - create
      - delete
      - get
      - list
      - patch
      - update
      - watch
  - apiGroups:
      - harvesterhci.io
    resources:
      - NodeNetworks
    verbs:
      - create
      - delete
      - get
      - list
      - patch
      - update
      - watch
```
3. In Rancher build another user called "test" password "testtesttest" - w/ Standard User & User-Base global permissions, additionally add the custom-role to that User "test"
4. As Admin in Rancher, create create a Harvester Cluster, adding "test" (Local) as a "Cluster Member"
5. In Harvester, enter in the Cluster Registration URL
6. In Rancher two windows ( one login as Admin ) - second (private window), login as "test" 
7. Navigate to Virtualization Management (as both users) -> Click the Cluster -> As Harvester Loads, click "support", you are re-directed

**Expected behavior**
Support link not to re-direct users, as some custom RBAC users in Rancher have limited Harvester permissions and "usually" get the Kubeconfig from "Support" link.  But also may generate support bundles, and this limits that ability to do so.  Plus additionally may want embedded Harvester Rancher or Longhorn link 

**Workaround for Kubeconfig**
1. Users, in Virtualization Management -> Harvester Clusters, clicking the three elipsis icon on the far right, allows user like "test" to download the Kubeconfig

**Workaround Unknown**
- Unknown workaround for a Rancher RBAC user (or admin) to generate a Support Bundle...?

**Environment**
 - Harvester ISO version: v1.3.2-rc2
 - Rancher v2.9.0, v2.9.1 
 - Underlying Infrastructure: qemu-kvm

**Additional context**

https://github.com/user-attachments/assets/371ee823-a60f-43df-9861-732fa5d17661

**Comment by irishgordo** [comment]
Note was using Rancher v2.9.0 -> but should be reproducible with v2.9.1 as well from reports

**Comment by irishgordo** [comment]
v2.8.5 Rancher & v1.3.1 Harvester do not encounter this:

https://github.com/user-attachments/assets/e91745fd-2056-4682-88f6-57c0b8d2f82b

**Comment by dfaltum** [comment]
I can also reproduce the problem.

**Comment by bpedersen2** [comment]
duplicates https://github.com/harvester/harvester/issues/6287

**Comment by a110605** [comment]
Hi @irishgordo , the fixed PR is merged to harvester-release-v1.3. Could you help verify ? thanks.
cc @bk201

**Comment by irishgordo** [comment]
Thanks so much for the fix @a110605 :+1: :smile: 

This looks great on:

- Harvester v1.3-c77ab6b-head:
    - v2.9.0 Rancher
    - v2.9.1 Rancher
 
With:
- UI Source: External
- UI-Index: https://releases.rancher.com/harvester-ui/dashboard/release-harvester-v1.3/index.html
- UI-Plugin-Index: https://releases.rancher.com/harvester-ui/plugin/harvester-release-harvester-v1.3/harvester-release-harvester-v1.3.umd.min.js


![Screenshot from 2024-09-03 11-54-02](https://github.com/user-attachments/assets/510a91d7-59f8-4ad2-9dcd-40fccc7b609f)
![Screenshot from 2024-09-03 11-25-16](https://github.com/user-attachments/assets/033f58ff-1a4d-4714-badb-631ad0a9be68)
![Screenshot from 2024-09-03 11-24-07](https://github.com/user-attachments/assets/c3d13b4c-3d8b-43fd-a881-317390c14a7b)
